packages/models-library/src/models_library/api_schemas_webserver/computations.py

```diff
@@ -100,6 +103,11 @@ class ComputationRunRestGet(OutputSchema):
     project_custom_metadata: dict[str, Any]
 
 
+class ComputationRunPathParams(BaseModel):
+    project_id: ProjectID
```

Comment:
Reuse  `ProjectsPathParams`. 
In the past, we have been using different names for the project id (i.e. `project_uuid` and `proejct_id`) precicely because we do not reuse

Same with the model_config. it is a general config for all `PathParams` models. It should be in a base class and applied everywhere